sklearn/decomposition/_pca.py

```diff
@@ -317,16 +317,16 @@ class PCA(_BasePCA):
     For svd_solver == 'arpack', refer to `scipy.sparse.linalg.svds`.
 
     For svd_solver == 'randomized', see:
-    `Halko, N., Martinsson, P. G., and Tropp, J. A. (2011).
+    Halko, N., Martinsson, P. G., and Tropp, J. A. (2011).
```

Comment:
if we delete `_ in the end of the references, then also delete ` in the beginning

Reply:
See my [comment](https://github.com/scikit-learn/scikit-learn/pull/22603#issuecomment-1052827016).

Reply:
Fixed. Also corrected minor errors in references.